R/FilteredData.R

```diff
@@ -181,25 +225,10 @@ FilteredData <- R6::R6Class( # nolint
       private$.check
     },
 
-    #' @description
-    #' Gets data attributes for a given dataset
-    #'
-    #' Sets and gets the data attribute on unfiltered data as it is never modified
-    #' as attributes.
-    #'
-    #' @param dataname (`character`) name of the dataset
-    #' @param attr (`character`) attribute to get from the data attributes of the dataset
-    #' @return value of attribute, may be `NULL` if it does not exist
-    get_data_attr = function(dataname, attr) {
-      private$check_data_varname_exists(dataname)
-      checkmate::assert_string(attr)
-      teal.data::get_attrs(self$get_filtered_dataset(dataname)$get_dataset())[[attr]]
-    },
-
```

Comment:
This fails as get_attrs is a method for TealDataset/TealData. These datasets no longer have attributes as varlabels can be obtained via get_varlabel. Only one occurrence downstream:
https://github.com/insightsengineering/teal.modules.general/pull/417/files